modules/microshift-audit-logs-config-intro.adoc

```diff
@@ -20,11 +20,18 @@ You can set fields in combination to define a maximum storage limit for retained
 
 {microshift-short} includes the following default audit log rotation values:
 
-* `maxFileSize`: 200Mb
-* `maxFiles`: 10 files
-* `maxFileAge`: 0, This value means that no default age limit is set.
-* `profile`: Default, This profile logs only metadata for read and write requests.
+.{microshift-short} default audit log values
+[cols="20%,20%,50%",options="header"]
+|===
 
-The maximum default storage usage for audit log retention is 2000Mb, provided that there are 10 or fewer files.
+|Audit log parameter|Default setting|Definition
+|`maxFileAge`:|`0`|How long log files are kept before automatic deletion. The default value means that a log file is never deleted based on age. This value can be configured.
+|`maxFiles`:|`10`|The total number of log files kept. By default, {microshift-short} retains 10 log files. The oldest is deleted when an excess file is created. This value can be configured.
```

Comment:
I personally like _retained_. To me it sounds better and it matches the following sentence.

```suggestion
|`maxFiles`:|`10`|The total number of log files retained. By default, {microshift-short} retains 10 log files. The oldest is deleted when an excess file is created. This value can be configured.
```

Reply:
The best word is probably "stored" now that I've thought about it _way too long_. But you're right, we do start with "rotation and retention."